client/src/redux/reducers/allocations.reducers.js

```diff
@@ -10,12 +10,18 @@ export const initialState = {
   errors: {},
   search: {
     results: [],
-    term: ''
+    term: '',
+    error: false
   },
   removingUserOperation: {
     userName: '',
     error: false,
     loading: false
+  },
+  addingUserOperation: {
```

Comment:
We should rename `addingUserOperation` and `removingUserOperation` to use simple present instead of present continuous conjugation, e.g. `addUserOperation`